extensions/src/test/java/org/apache/falcon/extensions/jdbc/ExtensionMetaStoreTest.java

```diff
@@ -57,28 +58,41 @@ public void init() {
     }
 
     @Test
-    public void dbOpertaions(){
+    public void testDBOperations(){
         //insert
-        stateStore.storeExtensionMetadataBean("test1", "test_location", ExtensionType.TRUSTED, "test_description");
+        stateStore.storeExtensionBean("test1", "test_location", ExtensionType.TRUSTED, "test_description");
 
         Assert.assertEquals(stateStore.getAllExtensions().size(), 1);
         //check data
-        ExtensionMetadataBean bean = stateStore.getDetail("test1");
+        ExtensionBean bean = stateStore.getDetail("test1");
         Assert.assertEquals(bean.getLocation(), "test_location");
         //delete
         stateStore.deleteExtensionsOfType(ExtensionType.TRUSTED);
         Assert.assertEquals(stateStore.getAllExtensions().size(), 0);
+
+        stateStore.storeExtensionBean("test2", "test_location", ExtensionType.CUSTOM, "test2_description");
```

Comment:
Nit : Better to have it in a separate test method testExtension and testExtensionJob